include/gatekeeper_gk.h

```diff
@@ -47,12 +49,22 @@ struct gk_config {
 	 * The fields below are for internal use.
 	 * Configuration files should not refer to them.
 	 */
+	unsigned int	   num_tunnels;
 	rte_atomic32_t	   ref_cnt;
 	struct gk_instance *instances;
 	struct net_config  *net;
+
+	/* Specify the Gatekeeper server IP addresses. */
+	struct in_addr     gk_inaddr;
+	struct in6_addr    gk_inaddr6;
+
+	struct ipip_tunnel_info *tunnels;
```

Comment:
I'd keep tunnel information on the LPM table, otherwise, it will require more effort to keep these independent data structs consistent. Especially when we add the dynamic configuration into the picture.

Since we still don't have the LPM table, I recommend to make the minimum compromise in the current patch, so adding the LPM table later is as easy as possible. In other words, add the minimum amount of code as need for this patch with relation to where the tunnels come from, because this code will mostly dropped later.